docs/reference/config.rst

```diff
@@ -0,0 +1,7 @@
+The Globus Compute Endpoint Configuration
+=========================================
+
+.. autoclass:: globus_compute_endpoint.endpoint.config.Config
```

Comment:
The `Config` object shape does not map directly to our required YAML schema, so I suggest we hold off on including this for the time being. See [ConfigModel.dict()](https://github.com/globus/globus-compute/blob/b26536412aa5bcc4f4b5ca09517b796fe68e322b/compute_endpoint/globus_compute_endpoint/endpoint/config/model.py#L160-L167) for context.

Reply:
hmm good point, fixed